examples/tcp-close/client.rs

```diff
@@ -328,4 +328,25 @@ impl TcpClient {
 
         Ok(())
     }
+
+    /// Issues a close() operation.
+    fn issue_close(&mut self, qd: QDesc) -> Result<()> {
+        if self.should_async_close {
+            let qt: QToken = self.libos.async_close(qd)?;
+
+            match self.libos.wait(qt, None) {
+                Ok(qr) if qr.qr_opcode == demi_opcode_t::DEMI_OPC_CLOSE && qr.qr_ret == 0 => {
+                    println!("async_close_succeeded");
+                    Ok(())
+                },
+                Ok(_) => anyhow::bail!("wait() should succeed with async_close()"),
+                Err(_) => anyhow::bail!("wait() should succeed with async_close()"),
```

Comment:
Isn't it valid to fail with `ECONNRESET`?

Reply:
Done, handled ECONNRESET.